CHANGELOG.md

```diff
@@ -13,6 +13,12 @@ All notable changes to this project will be documented in this file.
 "### Security" in case of vulnerabilities.
 -->
 
+## [2.2.0] - 2020-03-25

```

Comment:
versie klopt niet, misschien dat je toch beter in package.json versie 2.2.0 gaat behouden, anders is heel de tagging en changelog verwarrend